TrashMob/client-app/src/lib/event-helpers.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * A utility & helper function for event
+ */
+
+import EventData from '@/components/Models/EventData';
+import moment from 'moment';
+
+export const isCompletedEvent = (event: EventData) => {
+    return moment(event.eventDate).isBefore();
```

Comment:
Should this be eventDate+eventdurationhours+eventdurationminutes?

Reply:
That is true, will fix